kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md

```diff
@@ -244,154 +244,151 @@ It is no doubt true that in most vratas brāhmaṇas are to be fed, but one shou
+Works on vratas
 
-When a person undertakes a vrata he must see to it that he finishes it; for dire consequences are laid down for him who gives it up through heedlessness. Chāgaleya declares115 'when
+ This shows that towards the end of the thirteenth century, when foreign invaders had seized the larger part of India and were ruthlessly razing temples to the ground and converting by force and fraud thousands to their faith, the men of the day, both learned and ignorant, were expending vast intellectual labour and huge sums of money on vratas, pilgrimages and śrāddhas, blissfully ignorant of and blind to the political and religious dangers that engulfed them all.
 
-114. It has to be remembered that several different vratas are men. tioned under the same name, also that different dames are given to the same vrata and that some antis, atsavas and snapas also are described in works on vratas.
+The vratakośa edited by M. M. Gopinath Kaviraja (in the Sarasvatibhavana series in 1929 ) lists 1622 vratas, but that list is unduly swollen and inflated for various reasons[^114]. For example, the Asṭakā-śrāddhas and Anvaśṭaka-śrāddhas are included and have nine numbers assigned to them. There are twelve numbers assigned to Anantavrata because it is performed for twelve months from Mārgaśīrṣa, but Hemādri (on vrata vol. II. pp. 667-671 ) treats it as one vrata. Upavāsavrata is given four numbers, the only reference given being Hemādri (vrata) II. p. 509 (only 11/2 lines from Matsya). Hemādri (vrata II. pp. 748-52, 757 and so on) describes _Ekabhakta_ twelve times in each month from Caitra to Phālguna and then on II. p. 863 refers to him who performs Ekabhaktavrata for one year. The Vratakośa gives 13 separate numbers (2 for Ekabhakta and 11 for Ekabhukta, for this latter form there being no authority of Hemādri who calls all the 13 Ekabhaktavrata). Grahaśānti is given five numbers. It is unnecessary to multiply examples. If properly sifted the number would be much smaller than 1000.
 
-115. गृहीतव्रताकरणे छागलेयः। यो गृहीत्वा व्रतं मोहासाचरेत्काममोहितः। जीवग्भवति
 
-E a: Tor Jr Hq. . P. 784, 7. fil. fat. p. 9, a p. 153 (quoting from प्राय. विवेक), नि. सि. P. 26 (quoting from मदनरत्न and reading
+[^114]: It has to be remembered that several different vratas are men. tioned under the same name, also that different dames are given to the same vrata and that some śantis, utsavas and sṇanas also are described in works on vratas.
```

Comment:
snānas

Reply:
[^114]: It has to be remembered that several different vratas are mentioned under the same name, also that different names are given to the same vrata and that some śantis, utsavas and snānas also are described in works on vratas.
Corrections made.

Reply:
[^114]: It has to be remembered that several different vratas are mentioned under the same name, also that different names are given to the same vrata and that some śantis, utsavas and snānas also are described in works on vratas.
Corrected.

Reply:
corrected